test/format/salvage.c

```diff
@@ -139,16 +139,14 @@ wts_salvage(void)
     /* Salvage, then verify. */
     wts_open(g.home, &conn, &session, true);
     testutil_check(session->salvage(session, g.uri, "force=true"));
-
-#if 0
-    wts_verify("post-salvage verify");
-#endif
+    wts_verify(conn, "post-salvage verify");
     wts_close(&conn, &session);
 
     /* Corrupt the file randomly, salvage, then verify. */
     if (corrupt()) {
         wts_open(g.home, &conn, &session, false);
         testutil_check(session->salvage(session, g.uri, "force=true"));
+        wts_verify(conn, "post-corrupt-salvage verify");
```

Comment:
I think this is deliberately removed by @luke-pearson 's change. Can you confirm we can do this again?

Reply:
I think that you removed it in #5704, as part of the WT-6260 changes.

Regardless: if it doesn't work, we have a problem. The metadata file is salvaged in some cases, and we should not create a metadata file we can't verify, that's not a good plan.

Reply:
I only comment it out. @luke-pearson removed it later in WT-6278

Reply:
I removed this line in WT-6266 as we currently are working under the assumption that if we intentionally corrupt a datafile, it is extremely unlikely that verify will succeed due to the issues discussed in that ticket. Please don't add this line back until we resolve the discussion in WT-6266.

Reply:
@luke-pearson 
> I removed this line in WT-6266 as we currently are working under the assumption that if we intentionally corrupt a datafile, it is extremely unlikely that verify will succeed due to the issues discussed in that ticket. 

@luke-pearson, as I understand it, WT-6266 is discussing metadata verification, where verifying the HS file causes retrieval of records that may not exist in the underlying file. I believe the line of code we're discussing verifies the individual file after it has been salvaged, and so should not have problems. Or am I misunderstanding something?

Reply:
Alright my mistake, certainly __wt_verify verifies the specific tree and doesn't perform the backwards traversal from the history store to the data store file. It still does verify the history store for each key in the data store file but that doesn't cause the issue I saw.